frontend/src/lib/components/PropertyFilters/PropertyFilters.tsx

```diff
@@ -60,7 +61,7 @@ export function PropertyFilters({
     }, [propertyFilters])
 
     return (
-        <div className="PropertyFilters" style={style}>
+        <div className={clsx('PropertyFilters', className)}>
```

Comment:
One snapshot for this component seems actually different, and I suppose this might be because of specificity clashing. The annoying thing with utility classes is that they all have the same specificity, so in this case `className` is not going to work the same way as `style` – `style` guarantees the override will take effect, and `className` doesn't guarantee that the `PropertyFilters`'s properties won't be used instead.

Reply:
Good catch! Turns out the only time `className` was used in the `<PropertyFilter` component was to set the margin to 0, which was never specified in the first place 🤷 

What was changing the snapshot was actually the `<PropertyFilterDisplay` component. I inlined the style because it was only used in a single place in the app, plus the snapshot (which then updated). Again the margin could be removed because it had no affect and replaced the `flex-col` with `flex-wrap` which really was the intended behaviour. Renders as expected now:
<img width="1208" alt="Screenshot 2023-11-06 at 18 21 29" src="https://github.com/PostHog/posthog/assets/6685876/c4dc87d9-5363-425d-8bbd-e015ddbc7c4c">